unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NavMap/NavmapToastView.cs

```diff
@@ -28,26 +30,22 @@ private void Awake()
 
         public void Populate(Vector2Int coordinates, MinimapMetadata.MinimapSceneInfo sceneInfo)
         {
-            if (sceneInfo == null)
-            {
-                gameObject.SetActive(false);
-                return;
-            }
+            bool sceneInfoExtists = sceneInfo != null;
```

Comment:
```suggestion
            bool sceneInfoExists = sceneInfo != null;
```

Reply:
If `sceneInfo` doesn't exist we shouldn't just early exit here?

Reply:
I removed the "close the toast when the scene info doesn't exist" logic, since the toast is super useful for the "jump in" button alone, let's use it even though the scene info is not loaded (i'm disabling the gameobject of the text data we don't have in that case)